contracts/SpokePool.sol

```diff
@@ -72,15 +72,22 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
         uint64 quoteTimestamp,
         address indexed originToken,
         address recipient,
-        address indexed sender
+        address indexed depositor
     );
     event FilledRelay(
         bytes32 indexed relayHash,
+        uint256 totalRelayAmount,
         uint256 newFilledAmount,
-        uint256 indexed repaymentChain,
         uint256 amountSentToRecipient,
```

Comment:
Instead of reporting how much they sent to the recipient, do you think we should report how much was added to the filled amount? We have the newFilledAmount, but not the diff. Seems like it might be easier (from a bot perspective) to compute how much the relayer is owed from that amount rather than the amountSent. Thoughts?

Reply:
Hmm I think you might be right. @chrismaree do you have instincts on this? The relayer would be able to compute `(1 + relayerFeePct) * fillIncreaseAmount`. Question though is whether they even would care about this new fill amount. Also, should we simply append the fill delta or replace the `amountSentToRecipient`?

Reply:
> The relayer would be able to compute `(1 + relayerFeePct) * fillIncreaseAmount`.

I may be wrong, but I think the formula for the repayment amount would be `(1 - lpFeePct) * fillIncreaseAmount`. Does that seem right to you?

Reply:
I guess you could compute `amountSentToRecipient` by calculating `(1 - relayerFeePct - lpFeePct) * fillIncreaseAmount`

Reply:
I replaced `amountToSend` with `filllAmountPreFees` and renamed the param to `fillAmount`